arelle/logging/handlers/LogHandlerWithXml.py

```diff
@@ -68,14 +68,19 @@ def propElts(properties: list[tuple[Any, Any, Any]], indent: str, truncateAt: in
                                     entityEncode(msg),
                                     refs))
     def recordToJson(self, logRec: logging.LogRecord) -> dict[str, Any]:
-        message = { "text": self.format(logRec) }
-        if logRec.args and isinstance(logRec.args, Mapping):
-            for n, v in logRec.args.items():
-                message[n] = str(v)
         return {"code": getattr(logRec, "messageCode", ""),
                 "level": logRec.levelname.lower(),
                 "refs": getattr(logRec, "refs", []),
-                "message": message}
+                "message": self.format(logRec),
```

Comment:
Do you know how the JSON format is used?  Is there a risk of this breaking someone by changing the meaning of the keys?  I wonder if `{"message": {"text": self.format(logRec), "args": self._buildJsonMessageArgs(logRec)}}` is marginally more compatible...

Reply:
I don't. From folks that I've talked to the json format does not appear to be widely used, but we don't have concrete usage details. I agree, your suggestion is better. I've updated the message format.

Reply:
It was identified by a service provider who sent a support email.